testdata/go/v4-alpha/memcached-operator/monitoring/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -0,0 +1,62 @@
+---
+rule_files:
+- /tmp/rules.verify
+
+group_eval_order:
+- memcached.rules
+
+tests:
```

Comment:
@assafad Hi, Please look at this example https://github.com/kubevirt/kubevirt/blob/11316e05586b95dd3b3b4898f19cb2a743031e2c/hack/prom-rule-ci/prom-rules-tests.yaml#L71.

You can add additional values and test the result in different eval_time and merge some of the tests together.

Reply:
Done